src/omp/backend.f90

```diff
@@ -147,10 +150,83 @@ subroutine transeq_z_omp(self, du, dv, dw, u, v, w, dirps)
       type(dirps_t), intent(in) :: dirps
 
       ! u, v, w is reordered so that we pass w, u, v
-      !call self%transeq_omp_dist(dw, du, dv, w, u, v, dirps)
+      call self%transeq_omp_dist(dw, du, dv, w, u, v, dirps)
 
    end subroutine transeq_z_omp
 
+   subroutine transeq_omp_dist(self, du, dv, dw, u, v, w, dirps)
+      implicit none
+
+      class(omp_backend_t) :: self
+      class(field_t), intent(inout) :: du, dv, dw
+      class(field_t), intent(in) :: u, v, w
+      type(dirps_t), intent(in) :: dirps
+      integer :: n_halo
+
+      call transeq_halo_exchange(self, u, v, w, dirps)
+
+      call transeq_dist_component(self, du, u, u, dirps%der1st, dirps%der1st_sym, dirps%der2nd, dirps)
+      call transeq_dist_component(self, dv, v, u, dirps%der1st_sym, dirps%der1st, dirps%der2nd_sym, dirps)
+      call transeq_dist_component(self, dw, w, u, dirps%der1st_sym, dirps%der1st, dirps%der2nd_sym, dirps)
```

Comment:
ah yeah. Indeed I can change that. Though, is there a reason we have to comply with the 79 columns limit? 
If it is mainly for aesthetics reasons, I'd argue that keeping single lines is a lot clearer than breaking it here for example.

Reply:
There is not really a reason we just put it in our in our style guide when we're trying to come up with something and fprettify was set to check this as well. I actually favour it because I normally write code in a default size terminal screen and going beyond 79 columns the line gets wrapped and to avoid this I don't go beyond that. Even if one doesn't write code on a terminal a quick edit when running the code on a cluster is what most people do I think and then wrapped lines can make things a little confusing. Also, when looking at the changes on github with the split view too long lines get wrapped which is again not ideal I believe. And these three lines in particular are so long that a horizontal scroll bar appears for me on github in the conversation tab here and even if my window is wide and can't see the whole line without scrolling.